frontend/src/components/FormEnter/FormEnter.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+
+import Input from '../Input/Input';
+import Button from '../Button/Button';
+
+import '../../../../styles/FormEnter.css';
+
+function FormEnter() {
+	const [activeRegistration, setActiveRegistration] = useState(true);
+	const [activeEnter, setActiveEnter] = useState(false);
+	const [activeRecovery, setActiveRecovery] = useState(false);
+	const [activeTitle, setActiveTitle] = useState(false);
+
+	const handleClickEnter = () => {
+		setActiveRegistration(false);
+		setActiveEnter(true);
+		setActiveRecovery(false);
+		setActiveTitle(false);
+	}
+
+	const handleClickRegistration = () => {
+		setActiveRegistration(true);
+		setActiveEnter(false);
+		setActiveRecovery(false);
+		setActiveTitle(false);
+	}
+
+	const handleClickRecovery = () => {
+		setActiveRegistration(false);
+		setActiveEnter(false);
+		setActiveRecovery(true);
+		setActiveTitle(true);
+	}
+
+	return (
+		<section className="form-enter">
+			<form action="" className="form-enter__container">
+				<div className="form-enter__content">
+					<div className={`form-enter__wrapper-title ${activeTitle ? 'visibility' : ''}`}>
+						<p
+							onClick={handleClickRegistration}
+							className={`form-enter__title ${activeRegistration ? 'form-enter__title_active' : ''}`}>
+							Регистрация
+						</p>
+						<p
+							onClick={handleClickEnter}
+							className={`form-enter__title ${activeEnter ? 'form-enter__title_active' : ''}`}>
+							Вход
+						</p>
+					</div>
+					<p className={`form-enter__title ${activeRecovery ? '' : 'visibility'}`}>Забыли пароль?</p>
+					<div className="form-enter__input-container">
+						<Input
+							inputType={'email'}
+							inputClassName={'input'}
+							inputPlaceHolder={"Email"}
+						/>
+
+						<div className={`form-enter__wrapper-input ${activeRecovery ? 'visibility' : ''}`}>
+
+							<Input
+								inputType={'password'}
+								inputClassName={'input input_type_password'}
+								inputPlaceHolder={"Введите пароль"}
+							/>
+						</div>
+						{/* <div className={`form-enter__wrapper-input ${activeRecovery ? 'visibility' : ''}`}>
+							<Input
+								inputType={'password'}
+								inputClassName={'input input_type_password'}
+								inputPlaceHolder={"Повторите пароль"}
+							/>
+						</div> */}
+					</div>
+					<div className="form-enter__button-container">
+						<Button
+							buttonText={'Зарегистрироваться'}
+							buttonClassName={`button indent button_disabled ${activeRegistration ? '' : 'visibility'}`}
+						/>
+						<Button
+							buttonText={'Войти'}
+							buttonClassName={`button indent ${activeEnter ? '' : 'visibility'}`}
+						/>
+					</div>
+					<Button
+						buttonText={'Восстановить пароль'}
+						buttonClassName={`button button__recovery ${activeRecovery ? '' : 'visibility'}`}
+					/>
```

Comment:
Так как кнопка у нас во всех состояниях одна и та же, можем использовать ее один раз, а buttonText менять в зависимости от стейта
```suggestion
	<Button buttonClassName={`form-enter__button`} 
		buttonText={formView === formState.registration
			? 'Зарегистрироваться'
			: formView === formState.entry
				? 'Войти'
				: 'Восстановить пароль'} />
```